components/Cases/CaseRecap/CaseRecap.jsx

```diff
@@ -0,0 +1,76 @@
+import PropTypes from 'prop-types';
+
+import ErrorMessage from 'components/ErrorMessage/ErrorMessage';
+import Spinner from 'components/Spinner/Spinner';
+import { useCaseByResident } from 'utils/api/cases';
+import { formatDate } from 'utils/date';
+import DICTIONARY from 'data/dictionary';
+
+const CaseRecap = ({ personId, recordId }) => {
+  const { data: record, error: recordError } = useCaseByResident(
+    personId,
+    recordId
+  );
+  if (recordError) {
+    return <ErrorMessage />;
+  }
+  if (!record) {
+    return <Spinner />;
+  }
+
+  const sortedData = [];
+
+  const isDate = (value) => {
+    if (!isNaN(new Date(value))) {
```

Comment:
This is only working for `date` what about all the other components? 
We should reuse the `Summary` component, otherwise we are duplicating the logic.